test/command/suite/select/filter/arithmetic_operation/slash/uint64_max_and_minus_float_one.expected

```diff
@@ -0,0 +1,38 @@
+table_create Values TABLE_NO_KEY
+[[0,0.0,0.0],true]
+column_create Values value COLUMN_SCALAR UInt64
+[[0,0.0,0.0],true]
+load --table Values
+[
+{"value": 18446744073709551615}
+]
+[[0,0.0,0.0],1]
+select Values   --output_columns 'value, value / -1.0'
+[
+  [
+    0,
+    0.0,
+    0.0
+  ],
+  [
+    [
+      [
+        1
+      ],
+      [
+        [
+          "value",
+          "UInt64"
+        ],
+        [
+          "value",
+          "UInt64"
+        ]
+      ],
+      [
+        18446744073709551615,
```

Comment:
`18446744073709551615` is casted to `-1` as int64_t.
So the result is -1/-1.0 = 1.0.